### PR TITLE
Grand Corridor (Cathedral F2)

### PR DESCRIPTION
**Pull request by vtange**
rather short compared to F1

### CODE REVIEW

File: QUEST_LV_0200.tsv [hunk 1]
```diff
@@ -4890,21 +4890,21 @@ QUEST_LV_0200_20150323_004888	Talk to the disciple, Ruodell
...
+QUEST_LV_0200_20150323_004895	$Priest Yosana told you that her research documents have disappeared and she suspects the demons have a role. Defeat the demons and retrieve her research documents.
+QUEST_LV_0200_20150323_004896	$You've found all of Priest Yosana's documents. Hand them over to Priest Yosana.
+QUEST_LV_0200_20150323_004897	$Retrieve %s
+QUEST_LV_0200_20150323_004898	$Talk to Priest Daram
+QUEST_LV_0200_20150323_004899	$Priest Daram needs your help. Talk to her.
+QUEST_LV_0200_20150323_004900	$Collect solvents by using the Scroll of Purification on the demons
+QUEST_LV_0200_20150323_004901	$Priest Daram told you that many solvents are needed to make the reagent. Use the scroll of purification on the demons and collect the solvents.
+QUEST_LV_0200_20150323_004902	$You've collected enough turbid solvents. Hand them over to Priest Daram.
+QUEST_LV_0200_20150323_004903	$Priest Daram is thinking about something. Talk to her.
+QUEST_LV_0200_20150323_004904	$Use with the reagent Daram created on the demons
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:4905]: to: $Use with the reagent Priest Daram created on the demons

File: QUEST_LV_0200.tsv [hunk 2]
```diff
@@ -4890,21 +4890,21 @@ QUEST_LV_0200_20150323_004888	Talk to the disciple, Ruodell
...
+QUEST_LV_0200_20150323_004905	$Daram told you that she successfully created a reagent, but she needs to experiment with it. Test the reagent on the demons.
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:4906]: to: $Priest Daram told you that she successfully created a reagent, but she needs to experiment with it. Test the reagent on the demons.

**Comment by Sunaries** [comment]
Good to merge after those 2 minor changes. :+1:

File: QUEST_LV_0200.tsv (hunk 1 above)
Comment (vtange) [inline QUEST_LV_0200.tsv:4905]: shouldn't it be: $Use the reagent Priest Daram created on the demons

I've changed mine to this ^

**Comment by Sunaries** [comment]
Good to merge.

File: QUEST_LV_0200.tsv (hunk 1 above)
Comment (Sunaries) [inline QUEST_LV_0200.tsv:4905]: It was like that in your PR. I just added 'Priest' to it. Anyway, I will add your edit in my PR.

**Comment by imcgames** [comment]
:+1:

**Pull request merged**